pages/api/exercises.ts

```diff
@@ -0,0 +1,37 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from "next";
+import query from "../../db";
+import NextCors from "nextjs-cors";
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  await NextCors(req, res, {
+    // Options
+    methods: ["GET", "HEAD", "PUT", "PATCH", "POST", "DELETE"],
+    origin: "*",
+    optionsSuccessStatus: 200, // some legacy browsers (IE11, various SmartTVs) choke on 204
+  });
+
+  if (req.method === "GET") {
+    const data = await query("SELECT * FROM exercises");
+    res.json(data.rows);
+  }
+
+  if (req.method === "POST") {
+    const el = req.body;
+    const data = await query(
```

Comment:
youll be receiving an array of objects for exercises, maybe use a for each function to iterate through each exercise? 
- refer back to the populate table function